src/containers/Tenant/Schema/SchemaTree/SchemaTree.tsx

```diff
@@ -19,10 +28,14 @@ interface SchemaTreeProps {
 
 export function SchemaTree(props: SchemaTreeProps) {
     const {rootPath, rootName, rootType, currentPath, onActivePathUpdate} = props;
-
     const dispatch = useTypedDispatch();
 
     const [_, setQueryMode] = useQueryModes();
+    const [open, setOpen] = React.useState(false);
+    const [parent, setParent] = React.useState<string>('');
+    const [child, setChild] = React.useState('');
+
+    const [createDirectory, {requestId}] = schemaApi.useCreateDirectoryMutation();
```

Comment:
Are you sure we should use `requestId` as a key to re-render?
If request fails, we don't have to repaint.
And when we get this id? What if we have slow network, and requestId is already here, but directory isn't created yet?